.github/workflows/auto-merge.yml

```
@@ -0,0 +1,51 @@
+name: CWL Viewer Build
+ 
+on:
+ push:
+   branches:
+     - main
+ pull_request:
+   branches:
+     - main
+ 
+jobs:
+ build:
+   name: Java
+   runs-on: ubuntu-20.04
+ 
+   steps:
+     - uses: actions/checkout@v2
+ 
+     - name: MongoDB in GitHub Actions
+       uses: supercharge/mongodb-github-action@1.3.0
+ 
+     - name: Restore dependency cache
+       uses: actions/cache@v2.1.4
+       with:
+         path: ~/.m2/repository
+         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+         restore-keys: |
+             ${{ runner.os }}-maven-
+ 
+     - name: Set up Java
+       uses: actions/setup-java@v2
+       with:
+         distribution: 'adopt'
+         java-version: '8'
+ 
+     - name: Install system packages
+       run: |
+         sudo apt-get -qq update
+         sudo apt-get install graphviz python3-pip
+         sudo pip install cwltool
+ 
+     - name: Build and test with Maven
+       run: |
+         ./mvnw test -B
+ 
+     - name: Coverage report
+       run: |
+         # upload coverage report
+         mvn prepare-package -DskipTests=true
+         mvn cobertura:cobertura coveralls:report -DrepoToken=${{ secrets.COVERALLS_TOKEN }} 
```

```suggestion
         mvn cobertura:cobertura coveralls:report -DrepoToken=${{ secrets.COVERALLS_TOKEN }} 
```

Consider implementing the suggestions at https://github.com/trautonen/coveralls-maven-plugin/issues/136#issuecomment-797322126

---

Okay